plugins/sudoers/def_data.in

```diff
@@ -372,6 +372,9 @@ log_servers
 log_server_timeout
 	T_TIMEOUT|T_BOOL
 	"Sudo log server timeout in seconds: %u"
+log_server_disable_keepalive
```

Comment:
I would really prefer to have this be log_server_keepalive and initialize it to true in init_defaults().  If someone doesn't want it enabled they can negate the option in sudoers.  That way the logic is more consistent between the client and server code.

Reply:
fixed, the logic has been negated, log_server_disable_keepalive has been changed to log_server_keepalive